.github/workflows/android.yml

```diff
@@ -42,109 +36,69 @@ jobs:
           fetch-depth: 0
           submodules: true
 
-      - name: Clone SDL Android repo
-        run: |
-          git clone --depth=100 https://github.com/pelya/commandergenius.git build.android
-          git -C build.android checkout ${SDL_ANDROID_REV}
+      - name: Set script permissions
+        working-directory: ./.github
+        run: chmod +x ./scripts/*.sh
```

Comment:
```
    steps:
      - uses: actions/checkout@v2
        with:
          fetch-depth: 1
          submodules: true
```

Reply:
i use this for the Clone dependencies step, there are is a lot of cloning in that and i don't think this action has a compact way of cloning many repositories with many versions